examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -246,16 +256,67 @@ export default class App extends PureComponent {
     this.setState({debugOptions});
   }
 
+  _renderMainOnMinimap() {
+    const {
+      tileset,
+      debugOptions: {minimapViewport}
+    } = this.state;
+    if (!minimapViewport) {
+      return null;
+    }
+    let data = [];
+    if (tileset) {
+      data = tileset.tiles
+        .map(tile => {
+          if (!tile.selected || !tile.viewportIds.includes('main')) {
+            return null;
+          }
+          const boundingVolume = tile.boundingVolume;
+          const cartographicOrigin = new Vector3();
+          Ellipsoid.WGS84.cartesianToCartographic(boundingVolume.center, cartographicOrigin);
+          let radius = boundingVolume.radius;
+          if (!radius && boundingVolume instanceof OrientedBoundingBox) {
+            const halfSize = boundingVolume.halfSize;
+            radius = new Vector3(halfSize[0], halfSize[1], halfSize[2]).len();
+          }
+          return {
+            coordinates: [cartographicOrigin[0], cartographicOrigin[1], cartographicOrigin[2]],
+            radius: boundingVolume.radius
+          };
+        })
+        .filter(tile => tile);
+    }
+    return new ScatterplotLayer({
```

Comment:
It seems strange to use a scattetplot but I suppose we can improve this later.

Reply:
Scatterplot is perfect to show MBS on the mini map. I don't know what to use for OBB so use Scatterplot as well.